_dev/apps/ui/src/components/monetization/monetization-banner-warning-update-module.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <b-alert
+    class="border border-warning d-md-flex align-items-md-center container"
+    variant="warning"
+    show
+  >
+    <div>
+      <h3 class="h3">
+        {{ $t('banner.monetization.bannerWarningTitle') }}
+      </h3>
+      <p>
+        {{ $t('banner.monetization.bannerWarningText') }}
+      </p>
+    </div>
+    <b-button
```

Comment:
I saw the same code on multiple components, maybe you can create a component to manage the module upgrade. It will be more useful because you can just import the component everywhere and it will avoid the duplicate code

Reply:
Indeed, I was thinking the same thing because the `AlertCmp.vue` and `AlertModuleDisabled.vue` components are similar. However, these components wrap the **bootstrap Alert component**, so I don't think there's any point in creating a component of a component. You could maybe put the Alert components directly in the parent files, but that would make the files more complex. What do you think?

Reply:
You can wrap it with the alert, the advantage of making it a component is that if you need to do a modification, you don't have to apply it to each parent component. If you need to make too many changes, it may not be wise to do so.

Reply:
I don't understand what the difference is between the new component and the bootstrap Alert component.